src/ui/CardTabTitleOtions/CardTabTitleOtions.jsx

```diff
@@ -0,0 +1,133 @@
+import { Options } from "../Options/Options";
+import "./CardTabTitleOptions.css";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+const BASE_URL = "https://bootcamp-api.codeit.kr/api";
+
+export function CardTabTitleOptions({ links }) {
+  const [tabList, setTabList] = useState([]);
+  const [clickTabId, setClickTabId] = useState(0);
+  const [clickTabAll, setClickTabAll] = useState(true);
+  const [tabName, setTabName] = useState("전체");
+
+  // tab목록 가져오기
+  async function fetchTabList() {
+    try {
+      const response = await axios.get(`${BASE_URL}/users/1/folders`);
+      const { data } = response;
+      setTabList(data.data);
+    } catch (error) {
+      throw new Error(error);
+    }
+  }
+
+  useEffect(() => {
+    fetchTabList();
+  }, []);
+
+  // 탭들을 클릭시 작동 핸들러
+  const handleSelectTab = (e) => {
+    setClickTabAll(false);
+    setClickTabId(e.target.id);
+
+    const getTab = tabList.filter((tab) => tab.id == e.target.id);
+
+    setTabName(getTab[0].name);
```

Comment:
34라인에서 filter 함수를 find 함수로 변경하면 이 코드도 아래와 같이 변경되어야 하겠네요
```suggestion
    setTabName(selectedTab.name);
```